src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,80 @@
-export const Pagination = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage?: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage = 1,
+  onPageChange,
+}) => {
+  const totalPages = Math.ceil(total / perPage);
+  const pages = Array.from({ length: totalPages }, (_, i) => i + 1);
+
+  const isPageFirst = currentPage === 1;
+  const isPageLast = currentPage === totalPages;
+
+  const handleBackwardsClick = () => {
+    if (!isPageFirst) {
+      onPageChange(currentPage - 1);
+    }
+  };
+
+  const handleForwardsClick = () => {
+    if (!isPageLast) {
+      onPageChange(currentPage + 1);
+    }
+  };
+
+  return (
+    <>
+      <ul className="pagination">
+        <li className={classNames('page-item', { disabled: isPageFirst })}>
+          <a
+            data-cy="prevLink"
+            className="page-link"
+            href="#prev"
+            aria-disabled={isPageFirst}
+            onClick={handleBackwardsClick}
```

Comment:
Using an `href` with a hash can lead to unwanted scroll behavior. Consider preventing the default action in the click event handler.